newrelic/resource_newrelic_workload_test.go

```diff
@@ -122,6 +122,159 @@ func TestAccNewRelicWorkload_EntityScopeAccountsOnly(t *testing.T) {
 	})
 }
 
+////TODO changes
+func TestAccNewRelicWorkload_BasicOnly(t *testing.T) {
+	resourceName := "newrelic_workload.foo"
+	rName := acctest.RandString(5)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNewRelicWorkloadDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNewRelicWorkloadBasicConfigOnly(rName),
```

Comment:
We should add an update and import step for at least one of these tests. Here's an example - https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/resource_newrelic_alert_policy_channel_test.go#L16